spec/unit/lib_shared_spec.rb

```diff
@@ -17,6 +17,12 @@
       options = %w(option1 option2)
       expect(shared.option_list(name, options)).to eq '--name "option1" --name "option2"'
     end
+
+    it 'should return nil if nil parameter provided' do
+      name = nil
+      options = '42'
```

Comment:
I think the reverse of this would make more sense, so test that name is something and options is nil that way it verifies the test case we are trying to verify